app-frontend/src/app/components/pages/project/createAnalysis/index.html

```diff
@@ -1,71 +1,80 @@
+<div class="sidebar">
     <div class="sidebar-actions-group" ng-show="$ctrl.selected.size === 0">
-      <a class="btn btn-small btn-transparent"
-         ui-sref="project">
-        <i class="icon-caret-left"></i>Back
-      </a>
+        <a class="btn btn-tiny btn-transparent" ui-sref="project">
+            <i class="icon-caret-left"></i>Back
+        </a>
     </div>
     <div class="selected-actions-group" ng-show="$ctrl.selected.size > 0">
-      <rf-selected-actions-bar
-          checked="$ctrl.allVisibleSelected()"
-          on-click="$ctrl.selectAll()"
-          action-text="$ctrl.selectText"
-      >
-        <button class="btn btn-transparent"
-                ng-click="$ctrl.removeLayers()">
-          Remove
-        </button>
-      </rf-selected-actions-bar>
+        <rf-selected-actions-bar
+            checked="$ctrl.allVisibleSelected()"
+            on-click="$ctrl.selectAll()"
+            action-text="$ctrl.selectText"
+        >
+            <button class="btn btn-transparent" ng-click="$ctrl.removeLayers()">
+                Remove
+            </button>
+        </rf-selected-actions-bar>
     </div>
     <div class="sidebar-scrollable list-group">
-      <rf-list-item ng-repeat="layer in $ctrl.layerList track by layer.id">
-        <item-selector>
-          <rf-list-item-selector
-              id="layer.id"
-              selected="$ctrl.isSelected(layer.id)"
-              on-select="$ctrl.onSelect(id)"
-              color="layer.colorGroupHex"
-          ></rf-list-item-selector></item-selector>
-        <item-title><strong ng-attr-title="{{layer.name}}">{{layer.name}}</strong></item-title>
-        <item-date>{{layer.date | date}}</item-date>
-        <item-actions>
-          <rf-list-item-actions
-              actions="$ctrl.layerActions.get(layer.id)" >
-          </rf-list-item-actions>
-        </item-actions>
-      </rf-list-item>
+        <rf-list-item ng-repeat="layer in $ctrl.layerList track by layer.id">
+            <item-selector>
+                <rf-list-item-selector
+                    id="layer.id"
+                    selected="$ctrl.isSelected(layer.id)"
+                    on-select="$ctrl.onSelect(id)"
+                    color="layer.colorGroupHex"
+                ></rf-list-item-selector
+            ></item-selector>
+            <item-title>
+                <strong ng-attr-title="{{ layer.name }}">{{ layer.name }}</strong>
+            </item-title>
+            <item-date ng-if="layer.rangeStart && layer.rangeEnd">
+                {{ layer.rangeStart | date }} - {{ layer.rangeEnd | date }}
+            </item-date>
+            <item-actions>
+                <rf-list-item-actions actions="$ctrl.layerActions.get(layer.id)">
+                </rf-list-item-actions>
+            </item-actions>
+        </rf-list-item>
+    </div>
+</div>
+<div class="main page-content-container">
+    <div class="page-header wide">
+        <h3>Choose an analysis to run</h3>
```

Comment:
I think changing the wording here to `Choose a template to run` would make more sense, Since we use `template` for the pagination count component below.